src/main/resources/application.yml

```diff
@@ -3,5 +3,5 @@ server:
     enabled: true
     min-response-size: 1
   use-forward-headers: true
-logging:
-  level.org.apache.tomcat: "DEBUG"
\ No newline at end of file
+  tomcat.protocol-header: x-forwarded-proto
```

Comment:
Based on the Spring Boot documentation, this is unneeded. See: https://docs.spring.io/spring-boot/docs/current/reference/html/howto-embedded-servlet-containers.html#howto-use-tomcat-behind-a-proxy-server, the only thing that's needed is to set `server.use-forward-headers` to true. At least, that is my understanding.

Reply:
Correct, these two properties are not needed:
tomcat.protocol-header: x-forwarded-proto
tomcat.remote-ip-header: x-forwarded-for
Tested on OCP 3.6.